akka-remote/src/main/scala/akka/remote/artery/FlightRecorder.scala

```diff
@@ -208,12 +200,33 @@ private[remote] object FlightRecorder {
   val HiFreqEntryCountFieldOffset = 16
 }
 
+private[akka] object FlightRecorderExtension extends ExtensionId[FlightRecorder] with ExtensionIdProvider {
+
+  override def lookup(): FlightRecorderExtension.type = FlightRecorderExtension
+
```

Comment:
I don't intend to make this public. I want to reserve the right to change anything.